app/console.html

```diff
@@ -2,16 +2,21 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <meta http-equiv="Content-Security-Policy" content="default-src 'self'">
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' https://fonts.googleapis.com 'sha256-osjHfNDP4at4lg2rARZvLE6bzOq48Q1BhccVPpPKt1Q='; font-src 'self' https://fonts.gstatic.com">
         <link rel="stylesheet" href="../node_modules/bootswatch/dist/cyborg/bootstrap.min.css">
         <link rel="stylesheet" href="../node_modules/jquery.json-viewer/json-viewer/jquery.json-viewer.css">
         <link rel="stylesheet" href="console.css">
         <title>ElectronForce Console</title>
+        <style>
```

Comment:
This style should be moved to the style sheet at app/dashboard.css or a class from bootstrap should be used in it place.

Reply:
yes, moving style to the stylesheet is better. I moved it as you suggested. Actually I am new to open source and learning new skills. 

And thank you for adding the hacktoberfest-accepted label.